netty/src/main/java/io/grpc/transport/netty/NettyClientHandler.java

```diff
@@ -268,43 +240,42 @@ protected void onStreamError(ChannelHandlerContext ctx, Throwable cause,
    * Attempts to create a new stream from the given command. If there are too many active streams,
    * the creation request is queued.
    */
-  private void createStream(CreateStreamCommand command, ChannelPromise promise) {
-    // Add the creation request to the queue.
-    pendingStreams.addLast(new PendingStream(command, promise));
-
-    // Process the pending streams queue.
-    createPendingStreams();
+  private void createStream(CreateStreamCommand command, final ChannelPromise promise) {
+    Http2Connection connection = connection();
+    if (connection.isGoAway()) {
+      promise.setFailure(goAwayStatus().asException());
+      return;
+    }
+    Http2Connection.Endpoint<Http2LocalFlowController> local = connection.local();
+    // TODO: Send GO_AWAY if streamId overflows
+    final int streamId = local.nextStreamId();
+    final NettyClientStream stream = command.stream();
+    final Http2Headers headers = command.headers();
+    stream.id(streamId);
+    encoder().writeHeaders(ctx, streamId, headers, 0, false, promise)
+            .addListener(new ChannelFutureListener() {
+              @Override
+              public void operationComplete(ChannelFuture future) throws Exception {
+                if (future.isSuccess()) {
+                  // Attach the client stream to the HTTP/2 stream object as user data.
+                  Http2Stream http2Stream = connection().requireStream(streamId);
+                  http2Stream.setProperty(NettyClientStream.class, stream);
+                } else {
+                  stream.transportReportStatus(Status.fromThrowable(future.cause()), true,
+                          new Metadata.Trailers());
+                }
+              }
+            });
   }
 
   /**
    * Cancels this stream.
    */
   private void cancelStream(ChannelHandlerContext ctx, CancelStreamCommand cmd,
-      ChannelPromise promise) throws Http2Exception {
+                            ChannelPromise promise) throws Http2Exception {
```

Comment:
The previous wrapping was fine and more consistent with the rest of the file.

I personally find the previous wrapping better because it doesn't need realignment when things change visibility, return type, or name in addition to providing more horizontal space to reduce overall number of wrapped lines. That is just to say, keeping the formatting as-is should be generally preferred in order to prevent constant reformatting changes as each author reverts another's change.